packages/polaris-viz/src/hooks/useFunnelBarScaling.ts

```diff
@@ -0,0 +1,61 @@
+import {useCallback, useMemo} from 'react';
+import type {ScaleLinear} from 'd3-scale';
+
+// Threshold to determine if we should scale the segments, i.e if the smallest segment is less than 10% of the tallest segment
+export const SCALING_RATIO_THRESHOLD = 0.1;
```

Comment:
Perhaps these ratios could be props? These values are where we landed with UX as an initial take at the scaling feature

Reply:
Good idea having the scaling threshold as a prop, given that different funnel charts might need diff thresholds for scaling.

Reply:
We decided to be opinionated about the scaling feature and not expose it as a prop on the chart for now